client/src/components/screens/PatientSubmission/index.jsx

```diff
@@ -444,16 +444,15 @@ class PatientSubmissionScreen extends React.Component {
         }
         return true;
       case 1: {
-        // const checkIfEmptyValue = (array) => {
-        //   if (array) {
-        //     if (array.findIndex(element => !element) !== -1) {
-        //       return false;
-        //     }
-        //     return true;
-        //   }
-        //   return false;
-        // };
-
+        const checkIfEmptyValue = (array) => {
+          if (array) {
+            if (array.findIndex(element => !element) !== -1) {
+              return false;
+            }
+            return true;
+          }
+          return false;
```

Comment:
You can inline the function with `return array != null && array.findIndex(element => !element) === -1;`